sdk/search/Azure.Search.Documents/tests/DocumentOperations/VectorSearchTests.Embeddings.cs

```diff
@@ -0,0 +1,32 @@
009913024f, 0.0135633685f, 0.02570812f, -0.006405405f, -0.010293628f, -0.014592731f, -0.010319578f, 0.0020262876f, -0.022645982f, -0.0053544166f, 0.022005873f, 0.011712246f, -0.019012935f, 0.003739009f, 0.0016792021f, 0.0008217602f, -0.0036308826f, -0.0161411f, 0.0027615468f, -0.014981986f, 0.021590669f, 0.011617094f, -0.019255139f, 0.02577732f, 0.024064599f, -0.001567832f, 0.024791207f, -0.015622094f, -0.014748434f, -0.018199826f, 0.0042515276f, -0.008005675f, -0.007573169f, -8.433855E-05f, -0.0012045274f, 0.025984922f, 0.009740021f, -0.014990636f, -0.002906436f, -0.038095072f, -0.011617094f, -0.005877748f, 0.012681058f, -0.030292677f, -0.0021246825f, 0.039133087f, -0.023545593f, -0.043215938f, 0.0039249863f, 0.033372115f, 0.021988573f, -0.04598397f, -0.012663757f, 0.005389017f, -0.01448893f, -0.0049521867f, -0.0323687f, 0.009342116f, -0.012291803f, -0.0037498216f, 0.0020511567f, -0.016400604f, 0.007499643f, 0.01849393f, 0.012023649f, 0.0029583368f, 0.011521943f, 0.042385526f, -0.016019998f, 0.012802159f, 0.00403095f, 0.0027334339f, -0.0049262363f, 0.017300215f, 0.011573844f, 0.026071424f, -0.010207127f, -0.008836085f, -0.024877708f, 0.0070325374f, -0.0242203f, 0.034790732f, -0.018044123f, 0.0058215223f, 0.0001747592f, 0.024497105f, 0.0045499564f, -0.02652123f, 0.005553369f, 0.027213238f, 0.01288866f, -0.011054837f, -0.012784859f, 0.010933735f, -0.011132688f, 0.007746171f, 0.020742958f, 0.0049781366f, -0.018251726f, 0.004206115f, 0.0030361877f, -0.01124514f, -0.0060723755f, 0.0017808409f, 0.028770257f, 0.020068249f, 0.016080549f, -0.0065567815f, -0.009852473f, 0.016193f, -0.0036438578f, 0.026936434f, 0.0033173163f, 4.5176537E-05f, 0.014618682f, 0.012213952f, -0.01767217f, -0.006366479f, 0.0015894573f, -0.006297278f, -0.0082478775f, 0.0026101698f, 0.011089438f, 0.0319708f, 0.015077137f, 0.02167717f, -0.043215938f, 0.015734546f, 0.009757321f, -0.0002461226f, 0.0039206613f, 0.02169447f, -0.038510278f, -0.004822435f, 0.016651457f, 0.00963622f, 0.02252488f, -0.006522181f, 0.009324816f, -0.015777797f, 0.0109423855f, -0.0023376916f, -0.036157448f, 0.015094438f, 0.018269027f, 0.01847663f, -0.006535156f, 0.0026772083f, 0.009938974f, 0.021434966f, -0.019774146f, -0.00320919f, -0.0015224189f, -0.0006168608f, -0.0040612253f, -0.00422774f, 0.011729546f, 0.0028912984f, 0.034479327f, 0.006764384f, 0.018943734f, -0.018666932f, 0.0075385687f, 0.007374217f, -0.009835172f, 0.0047445837f, 0.013770971f, 0.010890486f, 0.0047402587f, 0.025587019f, 0.024704708f, -0.0162103f, 0.0031594518f, 0.008213277f, 0.0068681855f, -0.037368465f, -0.02986017f, -0.015639395f, 0.015950799f, 0.0120842f, 0.0031572892f, 0.0028588604f, -0.027524643f, 0.01616705f, -0.04273153f, 0.0034838307f, 0.023822395f, 0.0016381141f, 0.0006325391f, 0.02498151f, -0.0036092573f, -0.00966217f, 0.003948774f, -0.038198873f, 0.02738624f, 0.01599405f, 0.02567352f, 0.018839935f, 0.004848385f, 0.0052808905f, 0.002651258f, 0.0012856222f, 0.0109423855f, 0.011418141f, -0.031659395f, 0.0029777994f, 0.0012088525f };
```

Comment:
Are you sure it's `0.021434966f` and not `0.021434967f`?  :smile:  But also, this is a very nice way to setup these tests.